internal/platform/database/database.go

```diff
@@ -0,0 +1,8 @@
+package database
+
+// WasteStorage interface describes storage contract
+type WasteStorage interface {
+	GetWasteClass(s string) (int, error)
+	SetClassForWaste(i int, s string) error
```

Comment:
Лучше давать параметрам понятные имена, чтобы использующий интерфейс понимал, что нужно передать в метод.

```suggestion
	GetWasteClass(name string) (int, error)
	SetClassForWaste(idx int, name string) error
```